app/views/layouts/application.html.erb

```diff
@@ -25,10 +25,17 @@
       <a href="<%= root_path %>" class="logo">
     <% end %>
       <!-- mini logo for sidebar mini 50x50 pixels -->
-      <span class="logo-mini"><img src="/apple-icon-180x180.png">
+    <% if Flipper.enabled?(:new_logo) %>
+    <span class="logo-mini"><img src="/Just-Diaper.svg">
```

Comment:
I have absolutely no experience with accessibility and mobile devices - any idea if adding `alt` text here would help people using a screen reader on a mobile device?

Reply:
I have no idea either -- I imagine we'd add an alt tag and other goodness if we ended up using this example. Great catch :)